File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.py
class Solution:
    def average(self, salary: List[int]) -> float:
        Sum, count, maxElement, minElement = 0, 0, max(salary), min(salary)
        for sal in salary:
            if sal != maxElement and sal != minElement:
                Sum += sal
                count += 1
        return Sum / count

File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.cpp
class Solution {
public:
    double average(vector<int>& salary) {
        int Sum = 0, count = 0;
       int maxElement = *max_element(salary.begin(), salary.end()), minElement = *min_element(salary.begin(), salary.end());
        for (int &sal: salary) {
            if (sal != maxElement && sal != minElement) {
                Sum += sal;
                count++;
            }
        }
        return (double)Sum/(double)count;
    }
};